nautobot/core/ui/object_detail.py

```diff
@@ -378,25 +385,132 @@ def __init__(
         self,
         *,
         table_key,
+        table_title=None,
+        max_display_count=None,
+        include_fields=None,
+        exclude_fields=None,
+        add_button_route="default",
+        add_permissions=None,
+        related_field_name=None,
         body_wrapper_template_path="components/panel/body_wrapper_table.html",
         body_content_template_path="components/panel/body_content_table.html",
+        header_extra_content_template_path="components/panel/header_table.html",
+        footer_content_template_path="components/panel/footer_table.html",
```

Comment:
For self-consistency, should these maybe be named `header_extra_content_table.html` and `footer_content_table.html`?